src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -412,7 +412,11 @@ void cannotCreateTitleLevelRequestForUnknownInstance(String tlrFeatureStatus,
       .withRequesterId(patronId));
 
     assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
-    assertThat(postResponse.getJson(), hasErrorWith(hasMessage("Instance does not exist")));
+    if ("Title".equals(requestLevel)) {
```

Comment:
Try split this test in two: one for Title level request, one for Item level

Reply:
done